main.rb

```diff
@@ -0,0 +1,6 @@
+require './app'
+
+def main
+  app = App.new
+  app.run
+end
```

Comment:
It will be great to have a completed main method. Remember, your main method should be called at the end of the file.

- [ ] Also, have a way for the user to quit the app.
- [ ] Present the user with a list of options to perform.
- [ ] Let users choose an option.

For example, You can create a while loop, that lists all the options from one to seven.

- [ ] Also when i try running the app nothing happens on my terminal.

Reply:
Thank you very much for the observation, I will implement all changes  ASAP